### ISSUE TITLE
Channel profile picture in player lacks text alternative

### ISSUE THREAD

**Issue by arethsu** [Chocobozzz/PeerTube#7127]
### Description of issue

This is a violation of the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) [SC 1.1.1 Non-text Content (Level A)](https://www.w3.org/WAI/WCAG22/Understanding/non-text-content).

Images need to have an `alt` attribute. This should either contain a descriptive text alternative for assistive technology users or be deliberately empty to indicate that the image is purely decorative.

The profile picture image in the top left of the video widget has no `alt` attribute, which means that for example screen readers will announce its file name to users.

![No alt text attribute visible, screenshot of devtools.](https://github.com/user-attachments/assets/b48f9df8-8fac-443c-b208-c78559db249e)

### Steps to reproduce

1. Enter devtools, target channel profile picture, observe attributes

### Recommended solution

Ensure that the alt attribute is always set on photos, illustrations, and other meaningful images. Describe the subject concisely.

Code example:
```
<img alt="Child on skateboard, illustration." src="...">
```

Set an empty alt text on purely decorative images, or images where text adjacent to the image already describes all the content.

Code example:
```
<img alt="" src="coca_cola_50_cl"> 
<p>Coca cola, 50cl</p>
```

SVG images may need to be handled in a special way. See [Deque's guide about SVGs](https://dequeuniversity.com/rules/axe/4.2/svg-img-alt).

Our in-depth article on alt texts, written by a colleague who uses a screen reader, also contains information about SVGs - [Alt-texts: The Ultimate Guide](https://axesslab.com/alt-texts/)

### Additional information

* PeerTube instance:
  * URL: https://peertube-eu.howlround.com/
  * Version: 7.2.1

* Browser name, version, and platforms on which you could reproduce the bug: Safari 18.5, macOS 15.5

This issue was part of an accessibility audit carried out by [Axess Lab](https://axesslab.com/), under assignment of [HowlRound Theatre Commons](https://howlround.com/).

**Comment by Chocobozzz** [comment]
Thanks! Fixed by https://github.com/Chocobozzz/PeerTube/commit/d2064d873bc434ada862960fee3f99644681ea39
